elements/pfe-collapse/pfe-collapse-toggle.ts

```diff
@@ -38,12 +38,23 @@ export class PfeCollapseToggle extends LitElement {
 
   private logger = new Logger(this);
 
+  /**
+   * @readonly
+   * Use the toggle method to manually trigger expand.
```

Comment:
same as above. Also the doc block here should preferably describe the property itself and not just defer to another method